src/sentry/api/serializers/rest_framework/groupsearchview.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from sentry.models.savedsearch import SortOptions
+
+MAX_VIEWS = 50
+
+
+class ViewSerializer(serializers.Serializer):
+    id = serializers.CharField(required=False)
```

Comment:
what is id? should this be an int?

Reply:
This is the id of the view in the database. Most other endpoints cast ids to string when returning, and it seems like it has become precedent to expect a string in the input as well ([example](https://github.com/getsentry/sentry/blob/13975b06a59de20face9fd2d1ed9a3f61ccecc54/src/sentry/api/serializers/rest_framework/dashboard.py#L140))

Reply:
got it 👍🏼